browser/src/layer/vector/SVGGroup.js

```diff
@@ -9,8 +9,17 @@ L.SVGGroup = L.Layer.extend({
 
 	options: {
 		noClip: true,
+		manualDrag: window.touch.isTouchEvent,
```

Comment:
why we need this? can't we use directly `window.touch.isTouchEvent` as before?
I see no current code which uses that in a different way.